.github/workflows/ci.yml

```diff
@@ -42,8 +42,21 @@ jobs:
         with:
           ci-flags: "all"
 
-  build-all-providers:
-    name: Cargo check all-providers (current Rust stable & old compiler)
+  build-all-providers-msrv:
+    name: MSRV - Cargo check all-providers
+    runs-on: ubuntu-latest
+    if: ${{ always() }}
+    needs: [build-and-export-test-all-docker]
+    steps:
+      - uses: actions/checkout@v3
+      - name: Run the container to execute the test script
+        uses: ./.github/actions/ci_script
+        with:
+          ci-flags: "cargo-check"
+          rs-version: "1.66.0"
```

Comment:
Why not set this up as an user configurable input with 1.66.0 as the default version? That would allow custom jobs to test future versions or older versions.

Reply:
The default is and should be stable, as seen here: https://github.com/parallaxsecond/parsec/pull/737/files#diff-6b8e64fe03097f8751f9b85361f2037d351d24d2ed2994b3d23333ba4964e208R9.

So after this PR gets merged, every new PR will be tested with both stable and MSRV.

Is there any particular reason why we would want to add a new feature in the CI to allow the user to run with a specific compiler?